<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AirFlickMain</class>
 <widget class="QWidget" name="AirFlickMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Air Flick - Advanced Gesture Tracking Suite</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
  background-color: #18181b;
  color: #f4f4f5;
  font-family: 'Segoe UI', 'SF Pro Text', Arial, sans-serif;
}

QLabel {
  color: #e4e4e7;
}

QPushButton {
  border-radius: 6px;
  padding: 8px 12px;
  font-weight: 600;
  border: none;
  min-height: 36px;
}

QPushButton:hover {
  opacity: 0.85;
}

QPushButton:pressed {
  opacity: 0.7;
}

QGroupBox {
  border: 1px solid #27272a;
  border-radius: 8px;
  margin-top: 14px;
  padding-top: 8px;
  font-weight: 600;
}

QGroupBox::title {
  subcontrol-origin: margin;
  left: 8px;
  padding: 0 5px;
}

QSlider::groove:horizontal {
  border-radius: 3px;
  height: 6px;
  margin: 2px 0;
  background-color: #27272a;
}

QSlider::handle:horizontal {
  background-color: #8b5cf6;
  border: none;
  border-radius: 6px;
  width: 16px;
  margin: -5px 0;
}

QComboBox {
  padding: 6px 8px;
  border-radius: 6px;
  background-color: #27272a;
  border: 1px solid #3f3f46;
  min-height: 30px;
}

QComboBox::drop-down {
  subcontrol-origin: padding;
  subcontrol-position: right;
  width: 20px;
  border-left: 1px solid #3f3f46;
}

QScrollBar:vertical {
  border: none;
  background: #27272a;
  width: 8px;
  border-radius: 4px;
}

QScrollBar::handle:vertical {
  background: #52525b;
  min-height: 30px;
  border-radius: 4px;
}

QScrollBar:horizontal {
  border: none;
  background: #27272a;
  height: 8px;
  border-radius: 4px;
}

QScrollBar::handle:horizontal {
  background: #52525b;
  min-width: 30px;
  border-radius: 4px;
}</string>
  </property>
  <layout class="QHBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>12</number>
   </property>
   <property name="leftMargin">
    <number>12</number>
   </property>
   <property name="topMargin">
    <number>12</number>
   </property>
   <property name="rightMargin">
    <number>12</number>
   </property>
   <property name="bottomMargin">
    <number>12</number>
   </property>
   <item>
    <widget class="QWidget" name="sidebarWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>280</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="sidebarLayout">
      <property name="spacing">
       <number>12</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="logoFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background: #27272a;
border-radius: 8px;
padding: 12px;</string>
        </property>
        <layout class="QVBoxLayout" name="logoLayout">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="leftMargin">
          <number>8</number>
         </property>
         <property name="topMargin">
          <number>8</number>
         </property>
         <property name="rightMargin">
          <number>8</number>
         </property>
         <property name="bottomMargin">
          <number>8</number>
         </property>
         <item alignment="Qt::AlignCenter">
          <widget class="QLabel" name="titleLabel">
           <property name="styleSheet">
            <string notr="true">font-size: 26px;
font-weight: 700;
color: #8b5cf6;</string>
           </property>
           <property name="text">
            <string>AIR FLICK</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignCenter">
          <widget class="QLabel" name="subtitleLabel">
           <property name="styleSheet">
            <string notr="true">font-size: 13px;
color: #a78bfa;
font-weight: 500;</string>
           </property>
           <property name="text">
            <string>Advanced Gesture Tracking Suite</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignCenter">
          <widget class="QLabel" name="versionLabel">
           <property name="styleSheet">
            <string notr="true">font-size: 12px;
color: #71717a;
font-weight: 500;</string>
           </property>
           <property name="text">
            <string>Version 2.0</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="operationGroup">
        <property name="title">
         <string>OPERATIONS</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QGroupBox {
  color: #8b5cf6;
  font-size: 13px;
  letter-spacing: 1px;
}</string>
        </property>
        <layout class="QVBoxLayout" name="operationLayout">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="leftMargin">
          <number>8</number>
         </property>
         <property name="topMargin">
          <number>20</number>
         </property>
         <property name="rightMargin">
          <number>8</number>
         </property>
         <property name="bottomMargin">
          <number>8</number>
         </property>
         <item>
          <widget class="QPushButton" name="startButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: #10b981;
font-size: 14px;
color: white;</string>
           </property>
           <property name="text">
            <string>START TRACKING</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>icons/play.png</normaloff>icons/play.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="stopButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: #ef4444;
font-size: 14px;
color: white;</string>
           </property>
           <property name="text">
            <string>STOP TRACKING</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>icons/stop.png</normaloff>icons/stop.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="calibrateButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: #3b82f6;
font-size: 14px;
color: white;</string>
           </property>
           <property name="text">
            <string>CALIBRATE</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>icons/calibrate.png</normaloff>icons/calibrate.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="settingsGroup">
        <property name="title">
         <string>SETTINGS</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QGroupBox {
  color: #8b5cf6;
  font-size: 13px;
  letter-spacing: 1px;
}</string>
        </property>
        <layout class="QVBoxLayout" name="settingsLayout">
         <property name="spacing">
          <number>8</number>
         </property>
         <property name="leftMargin">
          <number>8</number>
         </property>
         <property name="topMargin">
          <number>20</number>
         </property>
         <property name="rightMargin">
          <number>8</number>
         </property>
         <property name="bottomMargin">
          <number>8</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="sensitivityLayout">
           <item>
            <widget class="QLabel" name="sensitivityLabel">
             <property name="styleSheet">
              <string notr="true">font-size: 13px;
font-weight: 500;</string>
             </property>
             <property name="text">
              <string>Mouse Sensitivity</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="sensitivityValue">
             <property name="styleSheet">
              <string notr="true">color: #8b5cf6;
font-weight: 600;</string>
             </property>
             <property name="text">
              <string>5</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QSlider" name="sensitivitySlider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>10</number>
           </property>
           <property name="value">
            <number>5</number>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksBelow</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="smoothnessLayout">
           <item>
            <widget class="QLabel" name="smoothnessLabel">
             <property name="styleSheet">
              <string notr="true">font-size: 13px;
font-weight: 500;</string>
             </property>
             <property name="text">
              <string>Movement Smoothness</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="smoothnessValue">
             <property name="styleSheet">
              <string notr="true">color: #8b5cf6;
font-weight: 600;</string>
             </property>
             <property name="text">
              <string>5</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QSlider" name="smoothnessSlider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>10</number>
           </property>
           <property name="value">
            <number>5</number>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksBelow</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="cameraLabel">
           <property name="styleSheet">
            <string notr="true">font-size: 13px;
font-weight: 500;
margin-top: 4px;</string>
           </property>
           <property name="text">
            <string>Camera Source</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cameraSelect">
           <property name="styleSheet">
            <string notr="true">padding: 6px;
margin-bottom: 8px;</string>
           </property>
           <item>
            <property name="text">
             <string>Default Camera</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Camera 1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Camera 2</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="settingsButton">
           <property name="styleSheet">
            <string notr="true">background-color: #52525b;
font-size: 13px;
color: white;</string>
           </property>
           <property name="text">
            <string>ADVANCED SETTINGS</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="helpGroup">
        <property name="title">
         <string>HELP</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QGroupBox {
  color: #8b5cf6;
  font-size: 13px;
  letter-spacing: 1px;
}</string>
        </property>
        <layout class="QVBoxLayout" name="helpLayout">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="leftMargin">
          <number>8</number>
         </property>
         <property name="topMargin">
          <number>20</number>
         </property>
         <property name="rightMargin">
          <number>8</number>
         </property>
         <property name="bottomMargin">
          <number>8</number>
         </property>
         <item>
          <widget class="QPushButton" name="tutorialButton">
           <property name="styleSheet">
            <string notr="true">background-color: #4f46e5;
font-size: 13px;
color: white;</string>
           </property>
           <property name="text">
            <string>TUTORIAL</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="aboutButton">
           <property name="styleSheet">
            <string notr="true">background-color: #8b5cf6;
font-size: 13px;
color: white;</string>
           </property>
           <property name="text">
            <string>ABOUT</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="mainContentLayout">
     <property name="spacing">
      <number>12</number>
     </property>
     <item>
      <widget class="QFrame" name="videoFrame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>300</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px solid #8b5cf6;
border-radius: 8px;
background-color: #27272a;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="videoLayout">
        <property name="leftMargin">
         <number>1</number>
        </property>
        <property name="topMargin">
         <number>1</number>
        </property>
        <property name="rightMargin">
         <number>1</number>
        </property>
        <property name="bottomMargin">
         <number>1</number>
        </property>
        <item>
         <widget class="QLabel" name="videoFeed">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">border: none;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="statusFrame">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>100</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #27272a;
border-radius: 8px;</string>
       </property>
       <layout class="QHBoxLayout" name="statusLayout">
        <property name="spacing">
         <number>12</number>
        </property>
        <property name="leftMargin">
         <number>12</number>
        </property>
        <property name="topMargin">
         <number>12</number>
        </property>
        <property name="rightMargin">
         <number>12</number>
        </property>
        <property name="bottomMargin">
         <number>12</number>
        </property>
        <item>
         <widget class="QFrame" name="gestureFrame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #18181b;
border-radius: 6px;
border: 1px solid #3f3f46;</string>
          </property>
          <layout class="QVBoxLayout" name="gestureFrameLayout">
           <property name="spacing">
            <number>2</number>
           </property>
           <property name="leftMargin">
            <number>10</number>
           </property>
           <property name="topMargin">
            <number>8</number>
           </property>
           <property name="rightMargin">
            <number>10</number>
           </property>
           <property name="bottomMargin">
            <number>8</number>
           </property>
           <item>
            <widget class="QLabel" name="gestureTitle">
             <property name="styleSheet">
              <string notr="true">font-size: 12px;
color: #a1a1aa;
font-weight: 500;
border: none;</string>
             </property>
             <property name="text">
              <string>CURRENT GESTURE</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="gestureOutput">
             <property name="styleSheet">
              <string notr="true">font-size: 16px;
color: #8b5cf6;
font-weight: 600;
border: none;</string>
             </property>
             <property name="text">
              <string>None</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="positionFrame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #18181b;
border-radius: 6px;
border: 1px solid #3f3f46;</string>
          </property>
          <layout class="QVBoxLayout" name="positionFrameLayout">
           <property name="spacing">
            <number>2</number>
           </property>
           <property name="leftMargin">
            <number>10</number>
           </property>
           <property name="topMargin">
            <number>8</number>
           </property>
           <property name="rightMargin">
            <number>10</number>
           </property>
           <property name="bottomMargin">
            <number>8</number>
           </property>
           <item>
            <widget class="QLabel" name="positionTitle">
             <property name="styleSheet">
              <string notr="true">font-size: 12px;
color: #a1a1aa;
font-weight: 500;
border: none;</string>
             </property>
             <property name="text">
              <string>CURSOR POSITION</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="positionOutput">
             <property name="styleSheet">
              <string notr="true">font-size: 16px;
color: #8b5cf6;
font-weight: 600;
border: none;</string>
             </property>
             <property name="text">
              <string>X: 0, Y: 0</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="calibrationFrame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #18181b;
border-radius: 6px;
border: 1px solid #3f3f46;</string>
          </property>
          <layout class="QVBoxLayout" name="calibrationFrameLayout">
           <property name="spacing">
            <number>2</number>
           </property>
           <property name="leftMargin">
            <number>10</number>
           </property>
           <property name="topMargin">
            <number>8</number>
           </property>
           <property name="rightMargin">
            <number>10</number>
           </property>
           <property name="bottomMargin">
            <number>8</number>
           </property>
           <item>
            <widget class="QLabel" name="calibrationTitle">
             <property name="styleSheet">
              <string notr="true">font-size: 12px;
color: #a1a1aa;
font-weight: 500;
border: none;</string>
             </property>
             <property name="text">
              <string>CALIBRATION STATUS</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="calibrationStatus">
             <property name="styleSheet">
              <string notr="true">font-size: 16px;
color: #8b5cf6;
font-weight: 600;
border: none;</string>
             </property>
             <property name="text">
              <string>Not Calibrated</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>